### PR TITLE
Support get topic applied policy for MaxUnackedMessagesPerSubscription

### PR DESCRIPTION
**Pull request by 315157973**
Master Issue: #9216

### Modifications
1) Fix the unackedMessagesExceededOnSubscription at the namespace level cannot be disabled
2) Added applied interface

### Verifying this change
test applied api : testMaxUnackedMessagesOnSubApplied
test priority : testMaxUnackedMessagesOnSubscriptionPriority

### CODE REVIEW

File: pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java [hunk 1]
```diff
@@ -2452,6 +2452,30 @@ public void setMaxUnackedMessagesPerSubscription(String namespace, int maxUnacke
         return asyncPostRequest(path, Entity.entity(maxUnackedMessagesPerSubscription, MediaType.APPLICATION_JSON));
     }
 
+    @Override
+    public void removeMaxUnackedMessagesPerSubscription(String namespace)
```
Comment (codelipenghui) [inline pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java:2456]: Do we need to also add a CLI command for this method?

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java [hunk 2]
```diff
@@ -2751,19 +2751,19 @@ protected void internalSetMaxSubscriptionsPerTopic(Integer maxSubscriptionsPerTo
         internalSetPolicies("max_subscriptions_per_topic", maxSubscriptionsPerTopic);
     }
 
-    protected void internalSetMaxUnackedMessagesPerSubscription(int maxUnackedMessagesPerSubscription) {
+    protected void internalSetMaxUnackedMessagesPerSubscription(Integer maxUnackedMessagesPerSubscription) {
         validateNamespacePolicyOperation(namespaceName, PolicyName.MAX_UNACKED, PolicyOperation.WRITE);
         validatePoliciesReadOnlyAccess();
-
+        if (maxUnackedMessagesPerSubscription != null && maxUnackedMessagesPerSubscription < 0) {
```
Comment (codelipenghui) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2757]: Currently, we use `null` to represent the absence of policy. How the users able to disable this policy for a specific namespace or topic?

For example, if the broker enabled the `maxUnackedMessagesPerSubscription` but we want to disable it in the namespace level or topic level.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java [hunk 3]
```diff
@@ -2731,7 +2731,7 @@ protected void internalSetMaxUnackedMessagesPerConsumer(int maxUnackedMessagesPe
         }
     }
 
-    protected int internalGetMaxUnackedMessagesPerSubscription() {
+    protected Integer internalGetMaxUnackedMessagesPerSubscription() {
```
Comment (codelipenghui) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2734]: If we want to introduce some change that will break the namespace level policy behavior, is it need to start a PIP to discuss it and it should be a highlight in the release note. @sijie What do you think

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java (hunk 2 above)
Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2757]: Just set it to 0, which is consistent with the description in broker.conf, but here it just cannot be set less than 0

File: pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java (hunk 1 above)
Comment (315157973) [reply pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java:2456]: Ok i will add

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java (hunk 3 above)
Comment (sijie) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2734]: It is better to have a PIP for this change. We should highlight it in the release notes.

**Comment by codelipenghui** [comment]
Sorry for the late response @315157973 Could you please help resolve the conflicts? I think after this PR merged, all the applied policy is done right? Please help update the state https://github.com/apache/pulsar/issues/9216 thanks.

**Review by codelipenghui — APPROVED** [review]

**Comment by 315157973** [comment]
> Sorry for the late response @315157973 Could you please help resolve the conflicts? I think after this PR merged, all the applied policy is done right? Please help update the state #9216 thanks.

Sorry, there are 3 left, I will finish it as soon as possible this week

**Pull request merged**